### ISSUE TITLE
A King being purchaseable complicates the game somewhat

### ISSUE THREAD

**Issue by sokorototo** [adam-mcdaniel/capitalist-chess#1]
Given that you can buy a King, you can have multiple Kings on the board. Do you lose when _any_ of your kings are checkmated, or do you have a primary King? Don't you think the idea of Chess is that while many things can happen on the deck, the main objective is a checkmate? If there are multiple ways to get a checkmate, doesn't that make the game chaotic, in a very non-chess manner? 
- Maybe there needs to be one King on the board,
- Creating a new King needs to be an expensive but strong defensive option that transforms the old King into a High-Rank piece, you get to promote a piece into a King, and the only requirement is that the King hasn't been in check for at least X moves.

**Comment by adam-mcdaniel** [comment]
This is a good point, I was considering removing this myself before I pushed the crate

Buying a King should be in response to a siege that can't be defended against. Maybe checkmate should only apply when all kings are guaranteed to be captured, but that would really draw out the game

I do like the idea of there only being one King on the board at a time, if players should be able to buy a King at all, maybe there should only be two so that the player can create one before the other is captured, so the game can continue

I think the *cleanest* solution would be to remove purchasing Kings entirely, that would be the most elegant and reduce the complexity of an already complex variant

**Comment by sokorototo** [comment]
Yeah, removing the ability to purchase Kings is the most straightforward solution. But you could have an election mechanic, that allows you to teleport the King to a safer position.
